packages/react-vapor/src/components/modal/examples/modalWindowExamples.tsx

```diff
@@ -0,0 +1,166 @@
+import * as loremIpsum from 'lorem-ipsum';
+import * as React from 'react';
+import {connect} from 'react-redux';
+
+import {ReactVaporStore} from '../../../../docs/ReactVaporStore';
+import {ExampleComponent} from '../../../../docs/src/components/ComponentsInterface';
+import {WithDirtyActions} from '../../../hoc/withDirty/withDirtyActions';
+import {IDispatch} from '../../../utils/ReduxUtils';
+import {Input} from '../../input/Input';
+import {Section} from '../../section/Section';
+import {Svg} from '../../svg/Svg';
+import {Tooltip} from '../../tooltip/Tooltip';
+import {ModalLoading} from '../loading/ModalLoading';
+import {closeModal, openModal} from '../ModalActions';
+import {ModalComposite} from '../ModalComposite';
+import {ModalCompositeConnected} from '../ModalCompositeConnected';
+
+export interface ModalConnectedExampleProps {
+    open?: (id: string) => void;
+    close?: (id: string) => void;
+    toggleIsDirty?: (id: string, isDirty: boolean) => void;
+}
+
+export const ModalWindowExamples: ExampleComponent = () => (
+    <Section title="Modal Window Examples">
+        <ModalCompositeExample />
+        <ModalCompositeExampleConnected />
+        <ModalLoadingExample />
+    </Section>
+);
+
+ModalWindowExamples.title = 'Modal Window';
+ModalWindowExamples.description =
+    'Modal windows appear in front of the main page and disable it while they are visible. They act as a zoom in on an element of the main page that allows additionnal interaction or configuration. They make possible for users to focus on their content whilst avoiding leaving the context from which they have been called.';
+
+// start-print
+
+const ModalCompositeExample: React.FunctionComponent = () => {
+    const [isOpened, setOpened] = React.useState(false);
+
+    return (
+        <Section level={3} title="A modal window component with a local state">
+            <div>
+                <button className="btn" onClick={() => setOpened(true)}>
+                    Open Modal
+                </button>
+                <ModalComposite
```

Comment:
I'm wondering if it's really useful to have a non-connected example

Reply:
If you think there is really no use cases where we will want to use an unconnected modal, I'll remove it. :+1: